content/docs/user-guide/networking/settings.md

```diff
@@ -0,0 +1,139 @@
+ | sv_serverSendRateMs     | Minimum number of milliseconds between each network update                           | 50 ms   || 
+
+### Replication Window Settings
+| Setting                            | Description                                                                               | Default | Notes |
+|------------------------------------|-------------------------------------------------------------------------------------------|---------|-------|
+| sv_ReplicateServerProxies          | Enable sending of ServerProxy entities to clients                                         | true    ||
+| sv_MaxEntitiesToTrackReplication   | The default max number of entities to track for replication                               | 512     ||
+| sv_MinEntitiesToReplicate          | The default min number of entities to replicate to a client connection                    | 128     ||
+| sv_MaxEntitiesToReplicate          | "The default max number of entities to replicate to a client connection"                  | 256     ||
+| sv_PacketsToIntegrateQos           | The number of packets to accumulate before updating connection quality of service metrics | 1000    |       |
+| sv_BadConnectionThreshold          | The loss percentage beyond which we consider our network bad                              | 0.25    ||
+| sv_ClientReplicationWindowUpdateMs | Rate for replication window updates                                                       | 300 ms  ||               
+| sv_ClientAwarenessRadius           | The maximum distance entities can be from the client and still be relevant.               | 500.0   ||
+
+### Rewind Settings
+| Setting                        | Description                                                                     | Default | Notes |
+|--------------------------------|---------------------------------------------------------------------------------|---------|-------|
+| sv_RewindVolumeExtrudeDistance | The amount to increase rewind volume checks to account for fast moving entities | 50      |       |
+
+### Editor Server Settings
+Settings to control how a server should be launched during the "play-in-editor" mode.
+
+| Setting               | Description                              | Default   | Notes |
+|-----------------------|------------------------------------------|-----------|-------|
+| editorsv_port         |                                          | 33450     |       | 
+| editorsv_serveraddr   | The address of the server to connect to. | LocalHost |       |
+| editorsv_process      |                                          | ""        |       |
+| editorsv_rhi_override |                                          | ""        | 
+| editorsv_enabled      |                                          | False     ||
+| editorsv_hidden       |                                          | False     || 
```

Comment:
If editorsv_launch is true, the Editor will automatically launch a sever upon entering game mode. Set editorsv_hidden to true if you want the launched server to be started as a background process without any visible window. This is different from editorsv_rhi_override=null, as that will start open a window, albeit without graphics. With editorsv_hidden, there will be no window.